proxy-node-shared/src/main/java/uk/gov/ida/notification/EidasSamlParserService.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ida.notification;
```

Comment:
Should this be in shared? It'll only ever be the gateway that uses it - maybe it should live there?